### ISSUE TITLE
[Store] Steam

### ISSUE THREAD

**Issue by vogler** [vogler/free-games-claimer#89]
Pretty much everything you can do in the browser is possible. Someone just has to implement it 😄
What set of games do you have in mind? I guess 'free to play' are not of interest.

This seems like a reasonable query:
https://store.steampowered.com/search/?sort_by=Reviews_DESC&as-hide=negative&maxprice=free&category1=998%2C21&specials=1&hidef2p=1&ndl=1

_Originally posted by @vogler in https://github.com/vogler/free-games-claimer/discussions/86#discussioncomment-5220406_

**Comment by EQUALIT-CG** [comment]
I would add free games Yes no Domes unless player enable it in and docker .env with steam_demos=1

But I'm thinking if they use this script to grabe all free games steam has to offer such as counter strike and everything 
PubG etc etc

**Comment by Primajin** [comment]
Have you tried https://steamdb.info/freepackages/ ?

**Comment by vogler** [comment]
No. Looks like a good temporary solution for people that are interested. Don't want to add another browser extension since I already use https://augmentedsteam.com.

**Comment by Primajin** [comment]
I wonder if the addon could be installed within the docker and thus the script could run that so then steam is also supported through piggybacking 😄

**Comment by vogler** [comment]
Playwright only directly supports [Chrome extensions](https://playwright.dev/docs/chrome-extensions), not Firefox: https://github.com/microsoft/playwright/issues/7297
Probably also more reliable to do it via Playwright than the extension, just needs to be implemented 😄
